API/contacts/AddContact.php

```diff
@@ -0,0 +1,32 @@
+<?php
+    ini_set('display_errors', 1);
+    ini_set('error_reporting', E_ALL);
+
+    require_once __DIR__ . '/../utils/JsonUtils.php';
+    require_once __DIR__ . '/../utils/ResponseSender.php';
+    require_once __DIR__ . '/../utils/RequestReciever.php';
+    require_once __DIR__ . '/../utils/ResponseCodes.php';
+
+    require_once __DIR__ . '/Contact.php';
+    require_once __DIR__ . '/ContactAPI.php';
+    
+    require_once __DIR__ . '/../database/Database.php';
+
+    $contact = RequestReciever::recievePOST(new Contact());
```

Comment:
by this you mean require or the statement? If it's statement, then it's kind of questionable because we receive something first and only then connect to database